Cargo.toml

```diff
@@ -17,6 +17,7 @@ byteorder = "1.3.4"
 libc = "0.2.68"
 regex = "1.3.5"
 thiserror = "1.0.19"
+bitflags = "*"
```

Comment:
Please put the specific version here. Since this is a library, there is no lockfile

Reply:
Good call, thanks!